packages/jupyterlab-manager/package.json

```diff
@@ -31,7 +31,8 @@
     "clean": "rimraf docs && rimraf lib",
     "docs": "typedoc --mode file --module commonjs --excludeNotExported --target es5 --moduleResolution node --out docs/ src",
     "lint": "tslint --project tslint.json --format stylish",
-    "prepublish": "npm run clean && npm run build"
+    "prepublish": "npm run clean && npm run build",
+    "watch": "tsc -w --project src"
```

Comment:
Do you need to run this when using `jupyter lab --watch`, or will it get in the way? I.e. do you end up with two processes trying to write to the same files?